lib/network.sh

```diff
@@ -52,19 +52,19 @@ prefixlen PROVISIONING_CIDR "$PROVISIONING_NETWORK"
 export PROVISIONING_CIDR
 export PROVISIONING_NETMASK=${PROVISIONING_NETMASK:-$(python -c "import ipaddress; print(ipaddress.ip_network(u\"$PROVISIONING_NETWORK\").netmask)")}
 
-network_address PROVISIONING_IP "$PROVISIONING_NETWORK" 1
-network_address CLUSTER_PROVISIONING_IP "$PROVISIONING_NETWORK" 2
+network_address PROVISIONING_IP "$PROVISIONING_NETWORK" 2
+#network_address CLUSTER_PROVISIONING_IP "$PROVISIONING_NETWORK" 3
```

Comment:
why do we need this?

Reply:
We want to use single IP .02 address.

Reply:
remove the commented line please

Reply:
I am reverting the changes and do the variable change only for the ironic-configmap.